Air Recovery Unit/Atrea RD5 Air Recovery.xml

```diff
@@ -9,23 +9,25 @@
     <ImportParameters>
       <Parameter>
         <Id>SlaveId</Id>
-        <Name>Slave Id</Name>
```

Comment:
meno tam malo zostat, len to s tymi zatvorkami

Reply:
<Name>{Slave Id}</Name>
takto?

Reply:
"<Name>Slave Id</Name>"

Reply:
tak ako to je v povodnom
<Name>${device_SlaveId}</Name>